int64 which_chess
---
float64[3] chess_location
